ballerina/init.bal

```diff
@@ -18,6 +18,7 @@ import ballerina/jballerina.java;
 
 function init() {
     setModule();
+    initializeHttpLogs();
```

Comment:
```suggestion
    initializeHttpLogs(traceLogConfig, accessLogConfig);
```
Can we do this and make `initializeHttpLogs` the external function? So we can get rid of `newHttpLogManager`